internal/server/http/handler.go

```diff
@@ -5,11 +5,13 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"time"
 )
 
 // ImposterHandler create specific handler for the received imposter
 func ImposterHandler(imposter Imposter) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(imposter.GetDelay())
```

Comment:
```suggestion
if imposter.Delay() > 0 {
  time.Sleep(imposter.Delay()
}
```

Reply:
With that avoid to call sleep even when time is 0

Reply:
I'm not against it.,just thought it doesn't make a lot difference. Thanks!